lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -36,11 +36,15 @@
 #include "mlir/IR/ImplicitLocOpBuilder.h"
 #include "mlir/IR/Threading.h"
 #include "mlir/Pass/Pass.h"
+#include "llvm/ADT/StringRef.h"
 #include "llvm/ADT/StringSet.h"
 #include "llvm/ADT/TinyPtrVector.h"
 #include "llvm/Support/Debug.h"
 #include "llvm/Support/FormatVariadic.h"
 #include "llvm/Support/Parallel.h"
+#include <algorithm>
+#include <mlir/IR/Attributes.h>
+#include <mlir/IR/BuiltinAttributes.h>
```

Comment:
Nits: Can you check if any of these are unnecessary?

Reply:
Wish we had Unused imports equivalent of Scala-fix

Reply:
To be fair - these are compile time includes hence not possible!

Reply:
(or is it? ) ---> anyway will manually do it for now!